src/lib/db.ts

```
@@ -0,0 +1,54 @@
+
+if (!MONGODB_URI) {
+  throw new Error('Please define the MONGODB_URI environment variable');
+}
+
+interface MongooseCache {
+  conn: typeof mongoose | null;
+  promise: Promise<typeof mongoose> | null;
+}
+
+declare global {
+  var mongoose: MongooseCache;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'mongoose'. Consider to delete it or rename it.

'mongoose' is defined here:


(lint/suspicious/noRedeclare)